app/src/utils/UtilsJSX.tsx

```diff
@@ -0,0 +1,19 @@
+import { Typography } from '@mui/material';
+import { IDialogContext } from '../contexts/dialogContext';
+
+/**
+ * Simple reusable method to make a snackbar appear with a string of your choice.
+ *
+ * @param message string to show
+ * @param context reference to current DialogContext
+ */
+export const setPopup = (message: string, context: IDialogContext) => {
```

Comment:
I would probably rename this to `setSnackbar(message, dialogContext)`, or something to that effect; `setPopup` makes me think of a dialog rather than a snackbar

Reply:
Done!